### PR TITLE
fix(android): strip xmlns definitions from child elements in AndroidManifest.xml

### PR DESCRIPTION
**Pull request by jquick-axway**
**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-27746

**Note:**
The damage has already been done to the 9.0.0 built modules "ti.map" and "ti.playservices" since this issue is already in their "AndroidManfiest.xml" files packaged within their AARs. We will need to rebuild the ticket's listed modules after merging this PR.

### CODE REVIEW

File: android/cli/lib/android-manifest.js [hunk 1]
```diff
@@ -67,7 +67,18 @@ class AndroidManifest {
...
 		if (this._xmlDomDocument && this._xmlDomDocument.documentElement) {
+			// Write XML content to string.
 			text += this._xmlDomDocument.documentElement + os.EOL;
+
+			// Remove all "xmlns:android=<url>"" namespace definitions from child elements.
+			// Google only allows it in root <manifest/> element or else a build/validation error will occur.
+			let startReplaceIndex = text.indexOf('<manifest');
+			if (startReplaceIndex >= 0) {
+				startReplaceIndex = text.indexOf('>', startReplaceIndex);
+				text = text.replace(/xmlns:android=".*?"/g, (match, p1) => {
```
Comment (garymathews) [inline android/cli/lib/android-manifest.js]: Minor, but since there are no capture groups the arguments should be `(match, offset) => {`

Comment (garymathews) [reply android/cli/lib/android-manifest.js]: Minor, the regex could be amended to remove the duplicate space:
```
/\sxmlns:android=".*?"/g
```

Comment (jquick-axway) [reply android/cli/lib/android-manifest.js]: Good idea. I'll go ahead and do it. Thanks.

**Review by garymathews — APPROVED** [review]
CR: **PASS** 👍

**Pull request merged**